Tests/MySQLTests/MySQLRowParserTests.swift

```diff
@@ -151,6 +151,15 @@ public class MySQLRowParserTests: XCTestCase {
 
     XCTAssertTrue(row["header1"] is String, "Type should be String")
   }
+
+  public func testParsesFieldWithBlobValueSetsTypeString() {
+    cRow[0] = "1".getUnsafeMutablePointer()
+    headers[0].type = MySQLFieldType.Blob
+
+    let row = parser.parse(row: cRow, headers: headers)
+
+    XCTAssertTrue(row["header1"] is String, "Type should be String")
+  }
```

Comment:
again, im unsure whether we should introduce an additional test when using a BLOB type as binary data